bioblend/_tests/TestGalaxyInvocations.py

```diff
@@ -94,6 +95,25 @@ def test_get_invocation_report(self):
             ret = self.gi.invocations.get_invocation_report_pdf(invocation_id, "report.pdf")
             assert ret is None
 
+    @test_util.skip_unless_galaxy("release_23.0")
+    def test_get_invocation_archive(self):
+        invocation = self._invoke_workflow()
+        payload = {
+            "model_store_format": "rocrate.zip",
+            "include_files": True,
+            "include_deleted": False,
+            "include_hidden": False,
+        }
+        self.gi.invocations.wait_for_invocation(invocation["id"])
+        with tempfile.TemporaryDirectory() as folder:
+            file = f"{folder}/temp.rocrate.zip"
+            response = self.gi.invocations.get_invocation_archive(invocation["id"], payload=payload)
+            with open(file, "bw") as archive:
+                for chunk in response.iter_content(chunk_size=8192):
+                    archive.write(chunk)
+            # Verify file is not empty
+            assert os.path.getsize(file) > 0, "Downloaded archive is empty"
```

Comment:
Maybe you can check if it's a Zip file with https://docs.python.org/3/library/zipfile.html#zipfile.is_zipfile ?

Reply:
good suggestion